docs/getting-started-monitoring.md

```diff
@@ -47,31 +47,34 @@ First, you need to prepare a Kubernetes cluster. If you use a **minikube** envir
    * Example
      ```yaml
      alertmanager:
-     
+
        service:
          type: LoadBalancer
-     
+
      ...
-     
+
      grafana:
-     
+
        service:
          type: LoadBalancer
          port: 3000
-     
+
      ...
-     
+
      prometheus:
-     
+
        service:
          type: LoadBalancer
-     
+
      ...
      ```
    * Note:
        * If you want to customize the Prometheus Operator deployment using Helm Charts, you need to set the following configuration for monitoring Scalar products.
            * The `serviceMonitorSelectorNilUsesHelmValues` and `ruleSelectorNilUsesHelmValues` must be set to `false` (`true` by default) to make Prometheus Operator detects `ServiceMonitor` and `PrometheusRule` of Scalar products.
 
```

Comment:
As I was making changes to the second second-level bullet, I realized this bullet needed similar changes for consistency.

```suggestion
       * If you want to customize the Prometheus Operator deployment by using Helm Charts, you'll need to set the following configurations to monitor Scalar products:
           * Set `serviceMonitorSelectorNilUsesHelmValues` and `ruleSelectorNilUsesHelmValues` to `false` (`true` by default) so that Prometheus Operator can detect `ServiceMonitor` and `PrometheusRule` in Scalar products.
```

Reply:
Sorry, I'm not familiar with the correct grammar of English, but I have one question.

Strictly, the `ServiceMonitor` and `PrometheusRule` are NOT components of Scalar products. In other words, Scalar products do NOT include these components in themselves. They are separated resources and created by Scalar Helm Chart as Kubernetes resources as follows.

```
+---[Kubernetes]-----------------------------------------------------------------------------------------------+
|                                                                                                              |
| +------------------------+                                                                                   |
| | ScalarDB Cluster (Pod) |                                                                                   |
| +------------------------+                                                                                   |
|                                                                                                              |
| +------------------------+                                                                                   |
| | ScalarDL Ledger  (Pod) |                                                                                   |
| +------------------------+                                                                                   |
|                                                                                                              |
| +------------------------+                                                                                   |
| | ScalarDL Auditor (Pod) |                                                                                   |
| +------------------------+                                                                                   |
|                                                                                                              |
| +---------------------------------------+                                                                    |
| | ServiceMonitor (for ScalarDB Cluster) | <--- Created by Scalar Helm Chart for ScalarDB Cluster monitoring. |
| +---------------------------------------+      But this ServiceMonitor is not component of Scalar products.  |
|                                                It is one of the Kubernetes resources.                        |
| +---------------------------------------+                                                                    |
| | PrometheusRule (for ScalarDB Cluster) | <--- Created by Scalar Helm Chart for ScalarDB Cluster monitoring. |
| +---------------------------------------+      But this PrometheusRule is not component of Scalar products.  |
|                                                It is one of the Kubernetes resources.                        |
|                                                                                                              |
| +---------------------------------------+                                                                    |
| | ServiceMonitor (for ScalarDL Ledger)  | <--- Created by Scalar Helm Chart for ScalarDL Ledger monitoring.  |
| +---------------------------------------+      But this ServiceMonitor is not component of Scalar products.  |
|                                                It is one of the Kubernetes resources.                        |
| +---------------------------------------+                                                                    |
| | PrometheusRule (for ScalarDL Ledger)  | <--- Created by Scalar Helm Chart for ScalarDL Ledger monitoring.  |
| +---------------------------------------+      But this PrometheusRule is not component of Scalar products.  |
|                                                It is one of the Kubernetes resources.                        |
|                                                                                                              |
| +---------------------------------------+                                                                    |
| | ServiceMonitor (for ScalarDL Auditor) | <--- Created by Scalar Helm Chart for ScalarDL Auditor monitoring. |
| +---------------------------------------+      But this ServiceMonitor is not component of Scalar products.  |
|                                                It is one of the Kubernetes resources.                        |
| +---------------------------------------+                                                                    |
| | PrometheusRule (for ScalarDL Auditor) | <--- Created by Scalar Helm Chart for ScalarDL Auditor monitoring. |
| +---------------------------------------+      But this PrometheusRule is not component of Scalar products.  |
|                                                It is one of the Kubernetes resources.                        |
|                                                                                                              |
+--------------------------------------------------------------------------------------------------------------+
```

In this case, I feel it would be better to use `of` or `for` instead of `in` to avoid confusions.

```diff
- Prometheus Operator can detect `ServiceMonitor` and `PrometheusRule` in Scalar products.
+ Prometheus Operator can detect `ServiceMonitor` and `PrometheusRule` of Scalar products.
```

```diff
- Prometheus Operator can detect `ServiceMonitor` and `PrometheusRule` in Scalar products.
+ Prometheus Operator can detect `ServiceMonitor` and `PrometheusRule` for Scalar products.
```

What do you think?
(If I miss some English grammar things, please correct me! 🙇‍♀️)

Reply:
@kota2and3kan I see. Thank you for the explanation! 

With that in mind, I think using `for` would be accurate:

```suggestion
       * If you want to customize the Prometheus Operator deployment by using Helm Charts, you'll need to set the following configurations to monitor Scalar products:
           * Set `serviceMonitorSelectorNilUsesHelmValues` and `ruleSelectorNilUsesHelmValues` to `false` (`true` by default) so that Prometheus Operator can detect `ServiceMonitor` and `PrometheusRule` for Scalar products.

```

Reply:
@josh-wong 
I see. Looks good!
Thank you for the update!